app/components/primer/flex_component.rb

```diff
@@ -32,6 +35,40 @@ class FlexComponent < Primer::Component
     DEFAULT_DIRECTION = nil
     ALLOWED_DIRECTIONS = [DEFAULT_DIRECTION, :column, :column_reverse, :row, :row_reverse].freeze
 
+    # @example 100|Default
+    #   <%= render(Primer::FlexComponent.new(bg: :gray)) do %>
+    #     <div class="p-5 border bg-gray-light">Item 1</div>
+    #     <div class="p-5 border bg-gray-light">Item 2</div>
+    #     <div class="p-5 border bg-gray-light">Item 3</div>
```

Comment:
Might we use the Box component for these `div`s?

Reply:
I tried using `FlexItem`'s previously but changed in https://github.com/primer/view_components/pull/190/commits/6da5b8380eac67cb64e09737d737dface5e5c96e because using plain ol divs solved the styling problem I brought up in https://github.slack.com/archives/CGJTTJ738/p1612458675188800

Reply:
c32bb94 updates it to use `Box`!